src/components/Login.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { GoogleLogin } from 'react-google-login';
+import { refreshTokenSetup } from '../utils';
+
+const clientId = '47074991686-f8t945ggrggrddm5mdvi9alkof42fj5r.apps.googleusercontent.com';
+
+function Login(props) {
```

Comment:
Not important but since the only use of props is the `onLogin` function, it would be nice if it was destructered in the parameter declaration, like `function Login({ onLogin })`